src/components/GisidaWrapper/index.tsx

```diff
@@ -339,39 +339,80 @@ class GisidaWrapper extends React.Component<GisidaProps, GisidaState> {
     /**  Deal with structures */
     const { structures } = this.props;
     if (structures) {
-      const structureLayer: FillLayerObj = {
-        ...fillLayerConfig,
-        id: STRUCTURE_LAYER,
-        paint: {
-          ...fillLayerConfig.paint,
-          'fill-color': GREY,
-          'fill-outline-color': GREY,
-        },
-        source: {
-          ...fillLayerConfig.source,
-          data: {
-            ...fillLayerConfig.source.data,
-            data: JSON.stringify(structures),
+      // console.log(structures.features.map(f => f.geometry && f.geometry.type));
+      const structurePolygons = { ...structures, features: [] as StructureGeoJSON[] };
+      const structurePoints = { ...structures, features: [] as StructureGeoJSON[] };
+      structures.features.forEach((structure: StructureGeoJSON) => {
+        const featureType = structure.geometry && structure.geometry.type;
+        switch (featureType) {
+          case POINT: {
+            structurePoints.features.push(structure);
+          }
+          default: {
```

Comment:
Is there a danger that we may end up pushing things that aren't "structures"?

Reply:
I think that we should assume that if the geoms are included in the Superset slice response that they are indeed structures.